draft-js-inline-toolbar-plugin/src/index.d.ts

```diff
@@ -28,7 +28,7 @@ export interface ToolbarChildrenProps {
 }
 
 export interface ToolbarProps {
-  children(externalProps: ToolbarChildrenProps): ReactNode;
+  children?: ComponentType<ToolbarChildrenProps>;
```

Comment:
I'm sure it's not component meant here. It's render prop. It should be just a function.

Reply:
It's the same but whit this syntax I'm able to include it as optional.

Reply:
It's not the same. ComponentType can be class too.
```js
children: (externalProps: ToolbarChildrenProps) => ReactNode
```

Reply:
Oh, thanks them, fixed and thanks for the info :)